buckets.tf

```diff
@@ -5,7 +5,7 @@ data "aws_kms_key" "aws_s3_key" {
 module "test_bucket" {
   source = "git@github.com:xerris/aws-modules.git//s3"
 
-  bucket        = "${var.env}-test-bucket-xerris"
+  bucket        = "${var.env}-project-terraform-state-dev"
```

Comment:
this should be a different bucket than state bucket

Reply:
Shall I put it as test-bucket-xerris-nithin ?